module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -29,9 +29,7 @@
 
 namespace VuFind\Recommend;
 
-use Laminas\Cache\Storage\Adapter\AbstractAdapter as CacheAdapter;
-use Laminas\Config\Config;
-use VuFind\Connection\LibGuides;
```

Comment:
Did you mean to remove these two lines?  Sorry can't test this morning.

Reply:
Yes, this was intentional. They are unused, but they were not deleted by php-cs-fixer because there are comments that contain the words Config and LibGuides, which confused it into thinking they might be valid includes.